client/my-sites/theme/controller.jsx

```diff
@@ -29,12 +28,10 @@ let themeDetailsCache = new Map();
 
 export function makeElement( ThemesComponent, Head, store, props ) {
 	return(
-		<ReduxProvider store={ store }>
-			<Head title={ props.title } description={ props.description } type={ 'website' }
-				canonicalUrl={ props.canonicalUrl } image={ props.image } tier={ props.tier || 'all' }>
-				<ThemesComponent { ...omit( props, [ 'title' ] ) } />
-			</Head>
-		</ReduxProvider>
+		<Head title={ props.title } description={ props.description } type={ 'website' }
+			canonicalUrl={ props.canonicalUrl } image={ props.image } tier={ props.tier || 'all' }>
+			<ThemesComponent { ...omit( props, [ 'title' ] ) } />
```

Comment:
Similarly, this is probably worth considering while shuffling things around. It's strange that we pass everything but `title` to me. Why not `title`? Why not skip `description` and `image` etc…

Anyway, the main reason I paused here is because we're hiding all of the props by passing `...this.props` instead of explicitly passing what the `ThemesComponent` expects. Maybe this is moot coming from a container/wrapper and it's okay to leave. Otherwise, maybe there's room to make the data more explicit.